### PR TITLE
TOC reorganization

### PR DESCRIPTION
**Pull request by meichstedt**

### CODE REVIEW

**Comment by meichstedt** [comment]
@sascala I've cherry-picked your changes and added some additions:

* renamed sections: _Using Marathon_ vs. _Administration_ vs. _Development_
* moved articles to the (most) appropriate  section
* restored alphabetical order (as before)
* fixed markdown metadata syntax

Can you please check if I missed something?

File: docs/_layouts/docs.html [hunk 1]
```diff
@@ -3,203 +3,221 @@
...
-        </a>
-      </li>
-    </ul>
-  </div>
-  <div class="col-sm-9">
-    {{ content }}
-  </div>
+    <div class="col-sm-3">
+
+        <!-- <h5>Overview: What is Marathon and why is useful?</h5> -->
```
Comment (meichstedt) [inline docs/_layouts/docs.html]: I'd remove this if we don't enable it and put some content in here.

**Comment by sascala** [comment]
Thanks for your changes, @meichstedt. I'm not sure about alphabetizing the TOC sections -- I tried to order the topics according to logical workflow (eg, install --> configure --> upgrade).

BTW the other changed files in this PR are TOC-related -- I created some separate troubleshooting pages and made a few other tweaks.

**Comment by sascala** [comment]
OK, in the interest of getting the revised TOC published, I'll table the alphabetical order question for the moment.

Assuming that @meichstedt 's Readiness Checks document is also ready for publication, could you please merge, @aquamatthias ?

**Comment by aquamatthias** [comment]
Thanks!

**Pull request merged**